content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -401,9 +403,13 @@ data:
 In this example, a second operation was run as an `Update` by the manager called `kube-controller-manager`. The update changed a value in the data field which caused the field's management to change to the `kube-controller-manager`.
 {{< note >}}If this update would have been an `Apply` operation, the operation would have failed due to conflicting ownership.{{< /note >}}
 
-### Merge Rules
+### Merge Strategy
+
+Through the merging strategy implemented with Server Side Apply, it is intended to provide a generally more stable object lifecycle.
+Server Side Apply tries to merge fields based on the fact who manages them instead of overruling just based on values.
+This way it is intended to make it easier and more stable for multiple actors updating the same object by causing less unexpected interference.
```

Comment:
If there is still time, a few nits:
line 322: Change `The` to `the`: If the Server Side Apply feature is enabled, The `PATCH` endpoint ...
line 327: remove: `in` or change the wording around feature-gates flag: ... you need to include in the `--feature-gates ServerSideApply=true` flag
line 408: Change:
The merging strategy, implemented with Server Side Apply, provides a generally more stable object lifecycle.

Reply:
Thanks @kbhawkey!
Done.